import-export-cli/cmd/addApi.go

```diff
@@ -0,0 +1,221 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	wso2v1alpha1 "github.com/wso2/k8s-apim-operator/apim-operator/pkg/apis/wso2/v1alpha1"
+	"github.com/wso2/product-apim-tooling/import-export-cli/box"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+var flagApiName string
+var flagSwaggerFilePath string
+var flagReplicas int
+var flagNamespace string
+
+const apiCmdLiteral = "api"
+const apiCmdShortDesc = "handle APIs in kubernetes cluster "
+const apiLongDesc = `Add, Update and Delete APIs in kubernetes cluster. JSON and YAML formats are accepted.
+available modes are as follows
+* kubernetes`
+const apiExamples = utils.ProjectName + " add/update " + apiCmdLiteral + ` -n petstore --from-file=./Swagger.json --replicas=3 --namespace=wso2`
+const addCmdLiteral = "add"
+const addCmdShortDesc = "Add an API to the kubernetes cluster"
+const addCmdLongDesc = `Add an API from a Swagger file to the kubernetes cluster. JSON and YAML formats are accepted.
+available modes are as follows
+* kubernetes`
+const addCmdExamples = utils.ProjectName + " " + apiCmdLiteral + " " + `-n petstore --from-file=./Swagger.json --replicas=3 --namespace=wso2`
```

Comment:
This should be apimcli add api (now it shows as apimcli api ). Shall we correct it?

Reply:
sure